app/assets/javascripts/discourse/app/routes/app-route-map.js

```diff
@@ -278,4 +278,8 @@ export default function () {
       this.route("show", { path: "/:token" });
     }
   );
+
+  this.route("wizard", function () {
+    this.route("step", { path: "/steps/:step_id" });
+  });
```

Comment:
This used to be conditionally added depending on whether the user is admin or not (since the route is added in the wizard bundle), this is no longer the case here. We may still be able to make this conditional here if we can get that bit of information into here?

In any case it will just fail to load the wizard from the server if a regular user tries to go there.